src/js/Controllers/UIController.js

```diff
@@ -30,6 +30,7 @@ import sdlController from './SDLController'
 import SubmenuDeepFind from '../Utils/SubMenuDeepFind'
 import ttsController from './TTSController';
 import { ValidateImages, AddImageValidationRequest, RemoveImageValidationResult } from '../Utils/ValidateImages'
+import TTSController from './TTSController'
```

Comment:
@YuriiYefimenko in line 31 it's been already imported TTSController. Please remove additional import